content/influxdb/v2.4/install.md

```diff
@@ -314,6 +328,20 @@ See InfluxDB [configuration options](/influxdb/v2.4/reference/config-options/) f
     If you do not move the `influxd` binary into your `$PATH`, prefix the executable
     `./` to run it in place.
 
+{{< expand-wrapper >}}
+{{% expand "<span class='req'>Recommended</span> – Set appropriate directory permissions" %}}
+
+To prevent unwanted access to data, we recommend setting the permissions on the influxdb `data-dir` to not be world readable.
+
+Example:
+
+```shell
+> chmod 0750 ~/.influxdbv2
+```
+
+{{% /expand %}}
```

Comment:
Move this into a single `expand-wrapper` block as explained in the other comment.